features/draft/download.yml

```diff
@@ -0,0 +1,14 @@
+draft_date: 2024-07-17
+name: download
+description: TODO
```

Comment:
```suggestion
description: The `download` attribute for `<a>` elements loads the target URL as a file to be saved, instead of navigating to it. An optional attribute value sets a default file name.
```